components/google-built-opentelemetry-collector/extension/oauth2clientauthextension/go.mod

```diff
@@ -4,57 +4,58 @@ go 1.23.0
 
 require (
 	github.com/stretchr/testify v1.10.0
-	go.opentelemetry.io/collector/component v1.35.0
-	go.opentelemetry.io/collector/component/componenttest v0.129.0
-	go.opentelemetry.io/collector/config/configopaque v1.35.0
-	go.opentelemetry.io/collector/config/configtls v1.35.0
-	go.opentelemetry.io/collector/confmap v1.35.0
-	go.opentelemetry.io/collector/confmap/xconfmap v0.129.0
-	go.opentelemetry.io/collector/extension v1.35.0
-	go.opentelemetry.io/collector/extension/extensionauth v1.35.0
-	go.opentelemetry.io/collector/extension/extensiontest v0.129.0
+	go.opentelemetry.io/collector/component v1.34.1-0.20250610090210-188191247685
```

Comment:
Running `make update-google-otel-components` in the root of the project should fix this stuff